dojo/jira_link/views.py

```diff
@@ -108,6 +108,7 @@ def webhook(request, secret=None):
                                 finding.active = False
                                 finding.mitigated = now
                                 finding.is_Mitigated = True
+                                finding.mitigated_by, created = User.objects.get_or_create(username='JIRA')
```

Comment:
if you don't use `created`make it explicit with `_`, no?
```suggestion
                                finding.mitigated_by, _ = User.objects.get_or_create(username='JIRA')
```

Reply:
good point